### PR TITLE
New Pending State

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch8780]

File: src/features/SendAssets/SendAssets.tsx [hunk 1]
```diff
@@ -186,7 +182,13 @@ function SendAssets({ location }: RouteComponentProps) {
 
       provider
         .sendRawTx(signedTx)
-        .then((payload) => dispatch({ type: sendAssetsReducer.actionTypes.SEND_SUCCESS, payload }));
+        .then((payload) => dispatch({ type: sendAssetsReducer.actionTypes.SEND_SUCCESS, payload }))
+        .catch((err) =>
+          dispatch({
+            type: sendAssetsReducer.actionTypes.SEND_ERROR,
+            payload: err.reason ? err.reason : err.message
```
Comment (Mrtenz) [inline src/features/SendAssets/SendAssets.tsx:189]:
```suggestion
            payload: err.reason ?? err.message
```

File: src/features/SendAssets/SendAssets.tsx [hunk 2]
```diff
@@ -199,6 +201,14 @@ function SendAssets({ location }: RouteComponentProps) {
       completeBtnText={translateRaw('SEND_ASSETS_SEND_ANOTHER')}
       wrapperClassName={`send-assets-stepper ${protectTxShow ? 'has-side-panel' : ''}`}
       basic={isFeatureActive('PROTECT_TX')}
+      onRender={(goToNextStep) => {
```
Comment (Mrtenz) [inline src/features/SendAssets/SendAssets.tsx:204]: Don't use arrow functions in props.

File: src/hooks/useGasForm/useGasForm.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  };
+
+  const handleGasLimitEstimation = async (network: Network, tx: Partial<ITxObject>) => {
+    setIsEstimatingGasLimit(true);
+    try {
+      const gas = await getGasEstimate(network, tx);
+      setFieldValue('gasLimitField', gas);
+      setGasEstimationError(undefined);
+    } catch (err) {
+      setGasEstimationError(err.reason ? err.reason : err.message);
```
Comment (Mrtenz) [inline src/hooks/useGasForm/useGasForm.ts:123]:
```suggestion
      setGasEstimationError(err.reason ?? err.message);
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/components/TransactionFlow/__tests__/TxReceipt.test.tsx [hunk 4]
```diff
@@ -1,24 +1,26 @@
...
+  fTxReceipt,
+  fTxReceiptEIP1559
 } from '@fixtures';
 import { translateRaw } from '@translations';
 import { ExtendedContact, ILegacyTxObject, ITxStatus, ITxType2Object } from '@types';
 import { bigify, noOp, truncate } from '@utils';
 
 import { constructSenderFromTxConfig } from '../helpers';
-import { TxReceiptUI } from '../TxReceipt';
+import { default as TxReceipt, TxReceiptUI } from '../TxReceipt';
```
Comment (Mrtenz) [inline src/components/TransactionFlow/__tests__/TxReceipt.test.tsx:23]:
```suggestion
import TxReceipt, { TxReceiptUI } from '../TxReceipt';
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**